hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSet.java

```diff
@@ -97,5 +106,10 @@ public long get(RuntimeMXBean runtimeMXBean) {
                     }
                 }
         );
+        registeredMetrics.add("runtime.uptime");
+    }
+
+    public static Set<String> getRegisteredMetricNames() {
```

Comment:
In HealthMetrics, the metric names are being duplicated. Is this what you like, duplicating the names? All I need is to get all the statistics that are registered for RuntimeMetricSet and OperatingSystemMetricSet.